Tests/TestCartPage.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using FluentAssertions;
+using Extensions;
+using Core;
+using System.Collections.Generic;
+
+namespace Tests
+{
+    [TestClass]
+    public class TestCartPage : TestBase
+    {
+        [TestMethod]
+        public void AddingOneDeletingProductFromCart()
+        {
+            HomePage.
+                MoveToWomenCatalog()
+                .AddProductAndProceedToCart(ConfigManager.IndexOfProductZero)
+                .PressTrashButton(ConfigManager.IndexOfProductZero)
+                .AllertPerform().Should().BeTrue();
+        }
+
+        [TestMethod]
+        public void AddingSomeDeletingProductFromCart()
+        {
+            HomePage
+                .MoveToWomenCatalog()
+                .AddTwoProductsAndProceedToCart()
+                .PressTrashButton(ConfigManager.IndexOfProductZero)
+                .NumberOfProducts().Should().Be(1);
+        }
+
+        [TestMethod]
+        public void AddingQuantityToProductInCart()
```

Comment:
You can maybe create an assertions class. The assertions class can receieve a double array like here and make sure that the prices are correct. Look into AndConstraint<> and how to make assertion classes!!